lib/api/dsl/methods.js

```diff
@@ -7,8 +7,202 @@ var
   q = require('q'),
   util = require('util'),
   geohash = require('ngeohash'),
+  units = require('node-units'),
```

Comment:
Nope, as long as it doesn't use deprecated methods.

I was just curious about this particular library and found convert-units, which seemed more recent and with what seemed to be a cleaner API. But this is a minor "issue".